charts/prometheus-postgres-exporter/Chart.yaml

```diff
@@ -13,3 +13,5 @@ keywords:
 maintainers:
 - email: gianrubio@gmail.com
   name: gianrubio
+- email: zanhsieh@gmail.com
```

Comment:
You'd also need to add yourself to `/.github/CODEOWNERS`

Reply:
done